doc/sphinx-guides/source/conf.py

```diff
@@ -225,7 +225,7 @@
 html_static_path = ['_static']
 
 html_js_files = [
-    'js/jquery-3.4.1.min.js',
+    'js/jquery-3.5.1.min.js',
```

Comment:
Unfortunately this is not how jQuery is updated. I thought adding a physical file would work back in #4219, but apparently that just added a second jQuery library, instead of replacing the default.